### DISCUSSION TITLE
Welcome to sidetree Discussions!

### DISCUSSION THREAD

**Discussion by nembal** [decentralized-identity/sidetree#1155] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nembal** [comment]
@OR13 it (discussions) is now available for Sidetree and Crypto Wg for testing.

**Comment by OR13** [comment]
@nembal thanks! These are useful way of moving helpful comments and questions out of actionable editorial or code work items. I hope we can use them to provide better support for folks with questions.
